ansible-cli.js

```diff
@@ -25,7 +25,7 @@ async function execute({
 
   const ansibleCommandParams = {
     playbookName: params.playbookName,
-    sshPassword: params.sshCredentials.password,
+    sshPassword: params.sshCredentials?.password,
```

Comment:
If `sshCredentials` is not an object, `ansibleCommandParams` will end up having `sshPassword` with value `undefined`.

Reply:
That's fine because on line 75 we check if `sshPasword` is defined, so it will not break anything if it's undefined here.